server/routes/companyRoutes.js

```diff
@@ -64,16 +64,19 @@ router.get('/profile/:companyId', (req, res) => {
 })
 
 router.post('/profile/:id', (req,res) => {
-  //grabs id from the url
-  const id = req.params.id
-  const body = req.body
-  body.company_id = id
-  db.addProfile(body)
+  const id = Number(req.params.id)
+  let body = req.body
+  body.companyId = id
+  body.lastUpdatedDate = getDate()
```

Comment:
see if this has been renamed to `timestamp`

Reply:
it has. changed